ompi/mpi/man/man3/MPI_Exscan.md

```diff
@@ -0,0 +1,144 @@
+# Name
+
+`MPI_Exscan`, `MPI_Iexscan` - Computes an exclusive scan (partial
+reduction)
+
+# Syntax
+
+## C Syntax
+
+```c
+#include <mpi.h>
+
+int MPI_Exscan(const void *sendbuf, void *recvbuf, int count,
+
```

Comment:
Watch these blank lines in the C bindings.  I stopped noting them and didn't mark them in every man page where I saw them -- you might want to check the generated .md files for blank lines in the C bindings.